docs/configuration.md

```diff
@@ -1222,6 +1222,19 @@ Apart from these, the following properties are also available, and may be useful
   </td>
   <td>2.2.1</td>
 </tr>
+<tr>
+  <td><code>spark.shuffle.accurateBlockSkewedFactor</code></td>
```

Comment:
To @timlee0119 and @JoshRosen , shall we remove `.internal()` from the configuration definition together?

https://github.com/apache/spark/blob/bdea09128760a5d36410cae2074f70a77e95c12b/core/src/main/scala/org/apache/spark/internal/config/package.scala#L1387-L1389

Reply:
In addition, the PR title looks wrong to me because we are touching `spark.shuffle.accurateBlockSkewedFactor` instead of `spark.sql.adaptive....`.

Reply:
Yeah, I would match the description

Reply:
Sorry for the typo, I've fixed the PR title and removed `internal()`